plugin/searchindex.vim

```diff
@@ -55,8 +59,10 @@ noremap! <Plug>SearchIndex <Nop>
 nnoremap <silent> <Plug>SearchIndex :call <SID>PrintMatches()<CR>
 
 " Remap search commands.
-nmap <silent> n n<Plug>SearchIndex
-nmap <silent> N N<Plug>SearchIndex
+exec 'nmap <silent> ' . g:searchindex_next_key .
+\ ' :normal! n<CR><Plug>SearchIndex'
+exec 'nmap <silent> ' . toupper(g:searchindex_next_key) .
```

Comment:
It will. `normal!` (notice the `!`) executes rhs as if no user mappings exist, so `n` there always corresponds to `next match`. I have `n` noremapped to `j`, and the plugin works.

Do I understand your argument right?